assets/css/refills/_badges.scss

```diff
@@ -0,0 +1,21 @@
+$small-font-size: 0.75em;
+$badges: (
+  "alert": #fff6bf,
+  "default": #999,
+  "error": #fbe3e4,
+  "notice": #e5edf8,
+  "success": #e6efc2,
+) !default;
+
+@each $badge-type, $color in $badges {
+  .badge-#{$badge-type} {
+    background-color: $color;
+    border-radius: $small-font-size * 5;
+    color: darken($color, 60%);
+    display: inline-block;
+    font-size: $small-font-size;
+    line-height: 1;
+    padding: 0.4em 1.2em;
```

Comment:
`0.4` should be written without a leading zero as `.4`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#leadingzero)